docs/manual/cluster-admin/how-to-add-and-remove-nodes.md

```diff
@@ -139,17 +139,17 @@ To remove nodes from the cluster, there is no need to modify `hosts.yml`.
 Go into `~/pai-deploy/kubespray/`, run
 
 ```bash
-ansible-playbook -i inventory/pai/hosts.yml remove-node.yml -b --become-user=root -e "node=a,b" -e "@inventory/pai/openpai.yml"
+ansible-playbook -i inventory/pai/hosts.yml remove-node.yml -b --become-user=root -e "node=worker-node-to-remove-0,worker-node-to-remove-1" -e "@inventory/pai/openpai.yml"
```

Comment:
change to `--limit`

Reply:
> change to `--limit`

Different from `add-nodes`,  `-e` is suggested to remove-nodes, in kubespray doc :  https://github.com/kubernetes-sigs/kubespray/blob/release-2.11/docs/getting-started.md#remove-nodes

Reply:
Thx, got it